documentation/docs/scenarios/TMS_Extension.md

```diff
@@ -74,4 +74,4 @@ steps:
 
 ### Parameters
 
-For a detailed description of the relevant parameters, see [tmsUpload](../../../steps/tmsUpload/).
+For a detailed description of the relevant parameters, see [tmsUpload](https://sap.github.io/jenkins-library/steps/tmsUpload/).
```

Comment:
Isn't it better to work with relative links when being inside one common bunch of documents? E.g. inside one homepage it is a common approach to refer to other elements on the same server / application via relative links.

The advantages when working with relative links is:

* it works also in case we have to move to another url. When working with absolute links - containing the host - that needs to be adjusted. 
* it is more easy to e.g. deploy the site to somewhere else (e.g. github.io/marcusholl/jenkinslibrary) and to check some details on the fork. Relative links will work in that case, absolute links points to outside the fork.

To my understanding the issue can be resolved by providing `../../steps/tmsUpload/` rather than `../../../steps/tmsUpload/`. With that we keep the relative link.